templates/paragraphs/paragraph--oe-text-feature-media.html.twig

```diff
@@ -0,0 +1,30 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'featured_media' type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{% set _content_classes = "bg-light p-3" %}
+{% set content = content.field_oe_plain_text_long %}
+
+{% if image %}
+  {% set imagestyle = {
+    '#theme':      'image',
+    '#uri':        image.src,
+    '#alt':        image.alt,
+  } %}
+{% endif %}
+
+{{ pattern('featured_media', {
+  'variant': variant,
+  'embedded_media': embedded_media,
+  'image': imagestyle|render,
+  'poster_image': poster_image,
+  'content': content,
```

Comment:
lets check for empty fields, see how it was done here https://github.com/openeuropa/oe_theme/blob/3.x/templates/paragraphs/paragraph--oe-text-feature-media.html.twig#L26